### DISCUSSION TITLE
Max width or height?

### DISCUSSION THREAD

**Discussion by TravisWittman** [hrynko/vue-pdf-embed#47] [Q&A] [answered]
I was wondering if there was a way to set a max width or height for the canvas?

**Comment by hrynko** [comment, marked as answer]
Hi @TravisWittman,

You can apply the `max-width` and `max-height` CSS properties in case you don't need annotation and text layers. Otherwise, these layers may not adhere to the canvas as their dimensions are absolute.

Here is an example of using `max-width` at component level (applies to all child canvases):

```vue
<template>
  <vue-pdf-embed
    disable-annotation-layer
    disable-text-layer
    :source="pdfSource"
  />
</template>

<script>
import VuePdfEmbed from 'vue-pdf-embed'

export default {
  components: {
    VuePdfEmbed,
  },
  data() {
    return {
      pdfSource: '<YOUR_SOURCE>',
    }
  },
}
</script>

<style>
.vue-pdf-embed {
  margin: auto;
  max-width: 320px;
}

.vue-pdf-embed canvas {
  width: 100% !important;
  height: auto !important;
}
</style>
```
